deploy/helm/elastic-agent/.gitignore

```diff
@@ -0,0 +1,2 @@
+Chart.lock
```

Comment:
I think Chart.lock can stay checked in. The [Helm docs](https://helm.sh/docs/helm/helm_dependency_build/) say `helm dependency build` should operate on an existing lock file.